modules/python/math.linear/source/math_linear.i

```diff
@@ -224,3 +237,1053 @@
         return returnVals;
     }
 }
+
+%extend math::linear::MatrixMxN<1, 1, double>
```

Comment:
Or... is it possible to do a `%extend` of the templated base class, add in these methods there, and then explicitly instantiate the templates via `%template` with that new extended class?